#!/usr/bin/env python
PACKAGE = "hand_control"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("z_max", double_t, 0, "The maximal altitude of the hand", 50., 0.)
gen.add("hue", double_t, 0, "The color hue of the hand", 0., 0., 360.)
gen.add("delta_hue", double_t, 0, "The tolerance of the hue filter", 20., 0., 180.)
gen.add("sat_min", double_t, 0, "The minimum color saturation of the hand", 0.1, 0., 1.)
gen.add("sat_max", double_t, 0, "The maximum color saturation of the hand", 1., 0., 1.)
gen.add("val_min", double_t, 0, "The minimum color value of the hand", 0.1, 0., 1.)
gen.add("val_max", double_t, 0, "The maximum color value of the hand", 1., 0., 1.)
exit(gen.generate(PACKAGE, "hand_control", "Filtre"))
